net_handler/udp_conn.c

```diff
@@ -5,6 +5,8 @@ int socket_fd = -1;
 struct sockaddr_in dawn_addr = {0};
 socklen_t addr_len = sizeof(struct sockaddr_in);
 
+uint64_t start_time;
```

Comment:
Can you add a few comments to this and all of the other global variables here detailing what they do?